src/hmr-server-engine.ts

```diff
@@ -38,10 +38,9 @@ export class EsmHmrEngine {
     return null;
   }
 
-  setEntry(sourceUrl: string, imports: string[], isHmrEnabled: boolean) {
+  setEntry(sourceUrl: string, imports: string[]) {
     const result = this.getEntry(sourceUrl, true)!;
     const outdatedDependencies = new Set(result.dependencies);
-    result.isHmrEnabled = isHmrEnabled;
```

Comment:
Ideally we should reset this to false and make the runtime notify again when accept gets called, just did not do it for the small poc